packages/SwingSet/src/vats/comms/delivery.js

```diff
@@ -147,10 +135,10 @@ export function makeDeliveryKit(state, syscall, transmit, clistKit, stateKit) {
     const delim1 = message.indexOf(':');
     assert(delim1 >= 0, X`received message ${message} lacks seqNum delimiter`);
     const seqNum = message.substring(0, delim1);
-    const remote = getRemote(state, remoteID);
-    remote.lastReceivedSeqNum += 1;
+    const remote = state.getRemote(remoteID);
+    remote.advanceReceivedSeqNum();
```

Comment:
tiniest of nits: this could return the new seqnum, instead of making the next line read it back out

Reply:
Easy enough...